docs/data-plane-configuration.md

```diff
@@ -0,0 +1,73 @@
+# Data Plane Configuration
+
+This document describes how to dynamically update the NGINX Gateway Fabric data plane configuration.
+
+## Overview
+
+NGINX Gateway Fabric offers a way to update the data plane configuration dynamically without the need for a
+restart. The data plane configuration is stored in the NginxProxy custom resource. This resource is created
+during the installation of NGINX Gateway Fabric.
+
+If using manifests, the default name of the resource is `nginx-proxy-config`. If using Helm, the default name
+of the resource is `<release-name>-proxy-config`. It is deployed in the same Namespace as the controller
+(default `nginx-gateway`).
+
+The control plane only watches this single instance of the custom resource. If the resource is invalid per the OpenAPI
+schema, the Kubernetes API server will reject the changes. If the resource is deleted or deemed invalid by NGINX
+Gateway Fabric, a warning Event is created in the `nginx-gateway` Namespace, and the default values will be used by
+the data plane for its configuration. Additionally, the control plane updates the status of the resource (if it exists)
+to reflect whether it is valid or not.
+
+### Spec
+
+| name | description                                        | type              | required |
+| ---- | -------------------------------------------------- | ----------------- | -------- |
+| http | HTTP defines the NGINX HTTP context configuration. | [http](#spechttp) | no       |
+
+### Spec.HTTP
+
+| name      | description                                    | type                        | required |
+| --------- | ---------------------------------------------- | --------------------------- | -------- |
+| telemetry | Telemetry defines the telemetry configuration. | [telemetry](#spectelemetry) | no       |
+
+### Spec.Telemetry
```

Comment:
Should this be `Spec.HTTP.Telemetry`?

Reply:
If so, would it need to be `Spec.HTTP.Telemetry.Tracing`? But I also see there are parts where we just reference `proxy.Tracing` without needing to go through the objects.

Reply:
> If so, would it need to be Spec.HTTP.Telemetry.Tracing

Yes, I've updated it 👍🏼 

> But I also see there are parts where we just reference proxy.Tracing without needing to go through the objects.

I think what you're referring to is when we're referencing the `graph` Tracing object?

Reply:
Yea, it in the `configuration.go` files I'm seeing the graph's `NginxProxy` directly reference `proxy.Tracing`, why is that able to do that?